README.md

```diff
@@ -1,3 +1,24 @@
+# Harvest4Us Version 0.1.0
+**Harvest4Us is an android application that connects local farmers to their community, making it easy for consumers to buy fresh produce.**
+
+## Release Notes
```

Comment:
nit: from the assignment on canvas, this should be a heading level 1 #. This specific version's release notes should begin with a level 2 # with the version number, and the level 3 # sections should just be titled Features, Bug Fixes, and Known Issues.

Also, I don't think the individual bullets should be bolded